helm/aws-resolver-rules-operator/templates/secret.yaml

```diff
@@ -2,9 +2,11 @@ apiVersion: v1
 stringData:
   credentials: |-
     [default]
-    aws_access_key_id: {{ .Values.aws.accessKeyID | quote }}
-    aws_secret_access_key: {{ .Values.aws.secretAccessKey | quote }}
-    region: {{ .Values.aws.region| quote }}
+    aws_access_key_id: {{ .Values.aws.accessKeyID | quote | required "setting aws.accessKeyID is required" }}
+    aws_secret_access_key: {{ .Values.aws.secretAccessKey | quote | required "setting aws.secretAccessKey is required" }}
+    {{- if .Values.aws.region }}
+    region: {{ .Values.aws.region | quote }}
+    {{- end }}
```

Comment:
Do we actually need the region? If it's an optional parameter here, maybe we can get rid of it and never pass it.